mack/__init__.py

```diff
@@ -315,12 +320,27 @@ def delta_file_sizes(delta_table: DeltaTable):
 
 def humanize_bytes(n: int) -> str:
     for prefix, k in (
-        ("PB", 1e15),
-        ("TB", 1e12),
-        ("GB", 1e9),
-        ("MB", 1e6),
-        ("kB", 1e3),
+            ("PB", 1e15),
```

Comment:
8 space indentation?

Reply:
This change I don't know how black does this.. I can't find any black settings to prevent it.